### DISCUSSION TITLE
加入对 last.fm 的支持

### DISCUSSION THREAD

**Discussion by MCSeekeri** [SPlayer-Dev/SPlayer#245] [想法 Ideas]
将播放记录，歌单等与 last.fm 同步。
虽然在线版本将转为维护，但一些离线音乐播放器也支持将本地的播放记录上传到 last.fm，因此希望能够加入这一功能。

另附 last.fm 的 [API 文档](https://www.last.fm/api)

**Comment by imsyy** [comment]
好的，会考虑支持
